<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>868</width>
    <height>185</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="layout">
     <item row="2" column="8">
      <widget class="QLabel" name="cloud1Label">
       <property name="text">
        <string>云组1</string>
       </property>
       <property name="buddy">
        <cstring>cloud1</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLineEdit" name="gust"/>
     </item>
     <item row="3" column="4">
      <widget class="QLineEdit" name="vis"/>
     </item>
     <item row="2" column="10">
      <widget class="QLabel" name="cloud3Label">
       <property name="text">
        <string>云组3</string>
       </property>
       <property name="buddy">
        <cstring>cloud3</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QGroupBox" name="typeGroup">
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QRadioButton" name="becmg">
          <property name="text">
           <string>BECMG</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QRadioButton" name="tempo">
          <property name="text">
           <string>TEMPO</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QLabel" name="weatherWithIntensityLabel">
       <property name="text">
        <string>天气现象1</string>
       </property>
       <property name="buddy">
        <cstring>weatherWithIntensity</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="9">
      <widget class="QLineEdit" name="cloud2"/>
     </item>
     <item row="0" column="3" colspan="2">
      <widget class="QGroupBox" name="prefixGroup">
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="2">
         <widget class="QCheckBox" name="at">
          <property name="text">
           <string>AT</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="fm">
          <property name="text">
           <string>FM</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="tl">
          <property name="text">
           <string>TL</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="11">
      <widget class="QLabel" name="cbLabel">
       <property name="text">
        <string>积雨云</string>
       </property>
       <property name="buddy">
        <cstring>cb</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QCheckBox" name="cavok">
       <property name="text">
        <string>CAVOK</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="wind"/>
     </item>
     <item row="4" column="9">
      <widget class="QCheckBox" name="nsc">
       <property name="text">
        <string>NSC</string>
       </property>
      </widget>
     </item>
     <item row="3" column="10">
      <widget class="QLineEdit" name="cloud3"/>
     </item>
     <item row="2" column="9">
      <widget class="QLabel" name="cloud2Label">
       <property name="text">
        <string>云组2</string>
       </property>
       <property name="buddy">
        <cstring>cloud2</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="6">
      <widget class="QComboBox" name="weatherWithIntensity"/>
     </item>
     <item row="4" column="8">
      <widget class="QCheckBox" name="skc">
       <property name="text">
        <string>SKC</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="period">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="periodLabel">
       <property name="text">
        <string>时段</string>
       </property>
       <property name="buddy">
        <cstring>period</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="gustLabel">
       <property name="text">
        <string>阵风</string>
       </property>
       <property name="buddy">
        <cstring>gust</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="windLabel">
       <property name="text">
        <string>风向风速</string>
       </property>
       <property name="buddy">
        <cstring>wind</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="8">
      <widget class="QLineEdit" name="cloud1"/>
     </item>
     <item row="2" column="4">
      <widget class="QLabel" name="visLabel">
       <property name="text">
        <string>能见度</string>
       </property>
       <property name="buddy">
        <cstring>vis</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="11">
      <widget class="QLineEdit" name="cb"/>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="nosig">
       <property name="text">
        <string>NOSIG</string>
       </property>
      </widget>
     </item>
     <item row="3" column="7">
      <widget class="QComboBox" name="weather"/>
     </item>
     <item row="2" column="7">
      <widget class="QLabel" name="weatherLabel">
       <property name="text">
        <string>天气现象2</string>
       </property>
       <property name="buddy">
        <cstring>weather</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>becmg</tabstop>
  <tabstop>tempo</tabstop>
  <tabstop>fm</tabstop>
  <tabstop>tl</tabstop>
  <tabstop>at</tabstop>
  <tabstop>period</tabstop>
  <tabstop>wind</tabstop>
  <tabstop>gust</tabstop>
  <tabstop>vis</tabstop>
  <tabstop>weatherWithIntensity</tabstop>
  <tabstop>cloud1</tabstop>
  <tabstop>cloud2</tabstop>
  <tabstop>cloud3</tabstop>
  <tabstop>cb</tabstop>
  <tabstop>skc</tabstop>
  <tabstop>nsc</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
